tensorboard/plugins/interactive_inference/utils/inference_utils.py

```diff
@@ -79,14 +79,11 @@ def convert_pattern_to_indices(pattern):
       pieces = [token.strip() for token in pattern.split(',')]
       indices = []
       for piece in pieces:
-        try:
-          if '-' in piece:
-            lower, upper = [int(x.strip()) for x in piece.split('-')]
-            indices.extend(range(lower, upper + 1))
-          else:
-            indices.append(int(piece.strip()))
-        except ValueError as e:
-          raise common_utils.InvalidUserInputError(e)
+        if '-' in piece:
+          lower, upper = [int(x.strip()) for x in piece.split('-')]
```

Comment:
Please add second argument to `split`. 
```py
# Does this blow up if piece contains more than one '-'?
lower, upper = [int(x.strip()) for x in piece.split('-',)]

# Kinda better
lower, upper = [int(x.strip()) for x in piece.split('-', 1)]
```

Reply:
added argument and made sure that with more than one - it will treat as a parse error. see tests